sections/language_model.tex

```diff
@@ -0,0 +1,14 @@
+\section{Language model}
+The \gls{lda} model gets poor retrieval performance when it not used in combination with another model \cite{yang2009topic}.
+In \cite{yang2009topic}, they describe various combinations of \gls{lda} and other models. 
+The language model, they describe is similar to a query likelihood model, which generates a probability of how likely a given document $d$ produces a given query $q$.
+To calculate this probability, they need to find the likelihood for each word in the query by using function:
+
+$$ P(w|d) = \frac{N_d}{N_d + \lambda} \cdot \frac{tf(w,d)}{N_d} + (1 - \frac{N_d}{N_d + \lambda}) \cdot \frac{tf(w,D)}{N_D} $$
+where $N_d$ is the number of word tokens in $d$ and $tf(w,d)$ is the word frequency of $w$ in $d$. $\lambda$ is a Dirichlet smoothing factor and is set to the average document length.
+When $D$ is used, it is referring the document collection.
```

Comment:
Eller "document corpus" eller lignende. Det ville være for konsistens i artiklen, men kan ikke huske hvad vi bruger.

Reply:
Hvad skal vi kalde det? document corpus, document collection?

Reply:
document corpus er fint for mig

Reply:
```suggestion
When $D$ is used, it is referring to the document corpus.
```